backend/instancer/api/account.py

```diff
@@ -336,6 +336,26 @@ def login() -> ResponseReturnValue:
     }
 
 
+@blueprint.route("/logout", methods=["POST"])
+def logout() -> ResponseReturnValue:
+    """Logout the current user
+
+    Requires body param login_token"""
+    try:
+        token = request.form["token"]
+    except KeyError:
+        return {"status": "missing_login_token", "msg": "Missing login token"}, 401
+
+    res = authentication.del_session(token)
+    if not res:
+        return {"status": "logout_failed", "msg": "Invalid token"}, 500
```

Comment:
don't forget to update the status as well. Try to avoid `invalid_token` though since that's the message used when the auth header is invalid - maybe `invalid_form_token`? There might be something better idk. Also should be a 400 error, not 500